src/tests/java/picard/analysis/CollectGcBiasMetricsTest.java

```diff
@@ -0,0 +1,217 @@
+package picard.analysis;
+
+import htsjdk.samtools.SAMFileHeader;
+import htsjdk.samtools.SAMFileWriter;
+import htsjdk.samtools.SAMFileWriterFactory;
+import htsjdk.samtools.SAMReadGroupRecord;
+import htsjdk.samtools.SAMRecord;
+import htsjdk.samtools.SAMRecordSetBuilder;
+import htsjdk.samtools.SAMTextHeaderCodec;
+import htsjdk.samtools.metrics.MetricsFile;
+import htsjdk.samtools.util.BufferedLineReader;
+import htsjdk.samtools.util.IOUtil;
+import org.testng.Assert;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+import picard.cmdline.CommandLineProgramTest;
+import picard.sam.SortSam;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Random;
+import static org.testng.Assert.assertEquals;
+
+/**
+ * Created by kbergin on 3/26/15 to test GcBias MultiLevel Collector.
+ */
+public class CollectGcBiasMetricsTest extends CommandLineProgramTest {
+    final static String sample1 = "TestSample1";
+    final static String sample2 = "TestSample2";
+    final static String readGroupId1 = "TestReadGroup1";
+    final static String readGroupId2 = "TestReadGroup2";
+    final static String readGroupId3 = "TestReadGroup3";
+    final static String platform = "ILLUMINA";
+    final static String library1 = "TestLibrary1";
+    final static String library2 = "TestLibrary2";
+    final static String library3 = "TestLibrary3";
+
+    final static File TEST_DIR = new File("testdata/picard/sam/CollectGcBiasMetrics/");
+    final File dict = new File(TEST_DIR, "Mheader.dict");
+
+    File tempDir;
+    File tempSamFile;
+    File samFile;
```

Comment:
Not used.  (intelliJ subtly colors these light grey to indicate their lack of use in the code).